frontend/src/components/Topics/Topic/Messages/MessagesTable.tsx

```diff
@@ -16,6 +18,33 @@ export interface MessagesTableProps {
   isFetching: boolean;
 }
 
+interface MessageData {
+  Value: string | undefined;
+  Offset: number;
+  Key: string | undefined;
+  Partition: number;
+  Headers: { [key: string]: string | undefined; } | undefined;
+  Timestamp: Date;
+}
+
+type DownloadFormat = 'json' | 'csv';
+
+function padCurrentDateTimeString(): string {
+  const now: Date = new Date();
+
+  const year: string = now.getFullYear().toString();
+  const month: string = (now.getMonth() + 1).toString().padStart(2, '0');
+  const day: string = now.getDate().toString().padStart(2, '0');
+  const hours: string = now.getHours().toString().padStart(2, '0');
+  const minutes: string = now.getMinutes().toString().padStart(2, '0');
+  const seconds: string = now.getSeconds().toString().padStart(2, '0');
```

Comment:
Is this padding necessary? Can you add an inline comment explaining why we are doing that if so?

Reply:
the padding is for more human readable format (e.g. 07:02:04)
will add a comment for that

Reply:
can't we use some time lib here? I believe we had some in use already (dayjs?)

Reply:
using date-fns now